loopd/perms/perms.go

```diff
@@ -34,30 +34,18 @@ var RequiredPermissions = map[string][]bakery.Op{
 	"/looprpc.SwapClient/LoopOutTerms": {{
 		Entity: "terms",
 		Action: "read",
-	}, {
-		Entity: "loop",
-		Action: "out",
 	}},
 	"/looprpc.SwapClient/LoopOutQuote": {{
 		Entity: "swap",
 		Action: "read",
-	}, {
-		Entity: "loop",
-		Action: "out",
 	}},
 	"/looprpc.SwapClient/GetLoopInTerms": {{
 		Entity: "terms",
 		Action: "read",
-	}, {
-		Entity: "loop",
-		Action: "in",
 	}},
 	"/looprpc.SwapClient/GetLoopInQuote": {{
 		Entity: "swap",
 		Action: "read",
-	}, {
-		Entity: "loop",
-		Action: "in",
```

Comment:
I wonder if there was a strong rationale on placing these here?

Reply:
See my comment here, https://github.com/lightninglabs/terminal-web/issues/662#issuecomment-1594284365. We have to find out :-)